registry.json

```diff
@@ -87,8 +87,8 @@
   "angular-tree-control": "github:wix/angular-tree-control",
   "angular-ui-grid": "github:angular-ui/bower-ui-grid",
   "angular-ui-layout": "github:angular-ui/ui-layout",
-  "angular-ui-router": "github:angular-ui/ui-router",
-  "angular-ui-router-extras": "github:christopherthielen/ui-router-extras",
+  "angular-ui-router": "github:angular-ui/angular-ui-router-bower",
+  "angular-ui-router-extras": "npm:ui-router-extras",
```

Comment:
I just realized this is wrong.  The package on npm and bower is named `ui-router-extras`, not `angular-ui-router-extras`